dgv/monitoring/digest.ipynb

```diff
@@ -64,36 +64,17 @@
             "metadata": {},
             "outputs": [],
             "source": [
-                "start_date = datetime.strptime(DATE_AIRFLOW, '%Y-%m-%d')\n",
+                "end_date = datetime.strptime(DATE_AIRFLOW, '%Y-%m-%d')\n",
                 "if(PERIOD_DIGEST == 'daily'):\n",
-                "    end_date = start_date + relativedelta(days=1)\n",
```

Comment:
So what's been happening is that `{{ ds }}` which should return `The DAG run’s logical date as YYYY-MM-DD` according to [the doc](https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html#:~:text=in%20version%202.2.-,%7B%7B%20ds%20%7D%7D,-str) actually seems to return the previous day, for example in the logs:
`[2024-01-05, 07:00:16 CET] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): publish_mattermost_daily> on 2024-01-04`
So daily was fine because we were looking at the day before for `start_date` and (wrongly) setting `end_date` one day further so we ended up with the right period 🙃 
I'm thinking about replacing all these `{{ ds }}` occurrences with `datetime.today()` so that we better know what's happening, what do you think?

Reply:
I think "logical date" is where the magic happened. It's ` the start of the data interval represented by a DAG run` (cf [doc](https://airflow.apache.org/docs/apache-airflow/stable/faq.html#what-does-execution-date-mean)), but since we're always struggling, we could prefer a datetime approach?